src/Settings/ApplicationSettings/ApplicationSettingsScreen.tsx

```diff
@@ -32,6 +31,20 @@ export const ApplicationSettingsScreen = () => {
   const installationId = useSelector(installationIdSelector)
   const dispatch = useDispatch()
   const {currency} = useCurrencyContext()
+  const {
+    theme: {color},
+  } = useTheme()
+  const language = useLanguage()
+
+  const [isBalanceHidden, setIsBalanceHidden] = useState(false) // TODO: temporary functionality before adding the real feature
```

Comment:
Temporary thing before implementing balance hiding feature (we need a ticket for that right?)